docs/core/preview3/tutorials/using-on-mac-vs-full-solution.md

```diff
@@ -0,0 +1,109 @@
+
+4. In the solution explorer, open the context menu for the **Dependencies** node and choose **Add Packages...**.
+
+5. Choose "nuget.org" as the Package source, and check **Json.NET**. Click **Add Package**. The package should now appear under **Dependencies/NuGet** and be automatically restored. [Json.NET](http://www.newtonsoft.com/json) is the recommended library and what our code will use to perform [JSON](http://www.json.org/) serialization and deserialization.
+
+   ![Adding the Json.NET dependency to the library project](./media/vsmacfull03.png)
+
+6. Rename the `MyClass.cs` file to `Thing.cs`. Also rename the class and constructor `Thing`. Add a method: `public int Get(int number) => Newtonsoft.Json.JsonConvert.DeserializeObject<int>($"{number}");`
+
+7. On the **Build** menu, choose **Build All**.
+
+   The solution should build without error.
+
+### Writing the test project
+
+1. In the solution explorer, open the context menu for the Golden solution node and choose **Add**, **Add New Project...**. In the new project dialog, choose **.NET Core / App** (and not **Library**), and then **Console Application (.NET Core)**. Click **Next**. Name the new project "TestLibrary" and click **Create**.
+
+   ![Adding the test project](./media/vsmacfull05.png)
+
+2. In the **TestLibrary** project, open the context menu for the **References** node and choose **Edit References...**. Check the Library project and click **OK**. This adds a reference to your library from the test project.
+
+   ![Referencing the library from the test project](./media/vsmacfull04.png)
+
+3. The testing framework that we'll use in this tutorial is [xUnit](https://xunit.github.io/). Right-click **Packages** under the solution explorer and choose **Add Packages...**. Search for xunit, check "xUnit.net" and click **Add Package** (the version you see may vary; the latest is usually the best choice).
+
+   ![Adding a reference to xUnit](./media/vsmacfull06.png)
+   
+   Do the same for `Microsoft.NET.Test.Sdk` and "xunit.runner.visualstudio". Those libraries enable the command-line test runner to work.
+
+4. Rename the `Program.cs` file to `LibraryTests.cs`, then open it and change the code to:
+
+    ```csharp
+    using Library;
+    using Xunit;
+    
+    public class LibraryTests
+        [Theory]
+        [InlineData(42)]
+        [InlineData(0)]
+        [InlineData(-1)]
+        public void ThingGetsObjectValFromNumber(int number)
+        {
+            Assert.Equal(number, new Thing().Get(number));
+        }
+    }
+    ```
+
+   You should now be able to build the solution.
+   
+5. At the time of writing, Visual Studio for Mac doesn't integrate NUnit tests to its built-in test runner. We will still be able to run tests, but from the command-line. Right-click the `TestLibrary` project, and choose **Tools / Open in Terminal** from the menu. In the command-line, enter `dotnet test`.
```

Comment:
At the time of writing -> Currently
We will still be able -> You are still able 
Use a greater-than symbol (>) to separate sequential steps: **Tools** > **Open in Terminal**